### PR TITLE
Inline the built files into a single HTML file for easy distribution

### PR DESCRIPTION
**Pull request by KyleMaas**
Builds a file in the dist directory called index-inlined.html which contains all needed assets to run ssb-browser-demo without any further requests to the server.  Basically consolidates the entire program into one easy-to-download, easy-to-host, easy-to-distribute HTML file.

### CODE REVIEW

**Comment by arj03** [comment]
I guess this is an alternative to the workbox PWA. It's nice that its just a single HTML file. Of course you the browser can't cache as much when you make changes, but maybe that is ok. I'm thinking it should maybe be a separate build command?

**Comment by KyleMaas** [comment]
Something like that?

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "human-time": "0.0.2",
+    "inline-source-cli": "^2.0.0",
```
Comment (arj03) [inline package.json:12]: this should be a dev dependency

File: package.json [hunk 2]
```diff
@@ -34,8 +35,9 @@
     "uglifyify": "^5.0.2"
   },
   "scripts": {
-    "build": "mkdir -p build && browserify -p esmify --full-paths ui/browser.js > build/bundle-ui.js && node write-dist.js",
-    "release": "mkdir -p build && browserify -g uglifyify -p esmify ui/browser.js > build/bundle-ui.js && node write-dist.js"
+    "build": "mkdir -p build && browserify -p esmify --full-paths ui/browser.js > build/bundle-ui.js",
```
Comment (arj03) [inline package.json:38]: This now is missing  && node write-dist.js

**Comment by arj03** [comment]
Almost :) Added two notes.

**Comment by KyleMaas** [comment]
How's that?

(Thanks for your patience with me.  I'm not used to all this newfangled npm-based build stuff.)

**Comment by arj03** [comment]
Perfect. Thanks. You are doing great. It's really nice to see someone come in and just contribute patches. They don't look like much because they are small, but I already love the new favicon :-)

**Comment by KyleMaas** [comment]
Thanks!  What I lack in capabilities, I try to make up for in enthusiasm.  :)

**Pull request merged**